src/utils/addons.ts

```diff
@@ -9,7 +9,9 @@ export interface Addons {
 	outro?: string;
 }
 
-function evalIfFn(strOrFn: string | (() => string | Promise<string>)): string | Promise<string> {
+function evalIfFn(
+	strOrFn: string | (() => string | Promise<string>) | undefined
```

Comment:
Type change. This function is only used within this file and is already handling the `undefined` case anyway.